android/app/src/main/kotlin/com/chicagoroboto/features/info/InfoFragment.kt

```diff
@@ -39,16 +46,25 @@ class InfoFragment : Fragment(R.layout.info) {
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
     binding = InfoBinding.bind(view)
+
+    binding.motionLayoutRoot.setTransitionListener(object : TransitionAdapter() {
+
+      override fun onTransitionStarted(motionLayout: MotionLayout?, startId: Int, endId: Int) {
+        val typefaceStyle = if (endId == R.id.expanded) Typeface.BOLD else Typeface.NORMAL
+        binding.toolbarText.setTypeface(binding.toolbarText.typeface, typefaceStyle)
+      }
+    })
+
     binding.list.apply {
       layoutManager = LinearLayoutManager(context, VERTICAL, false)
       adapter = this@InfoFragment.adapter
     }
 
-    binding.inset.doOnApplyWindowInsets { view, insets, initialState ->
+    binding.inset.doOnApplyWindowInsets { _, insets, _ ->
```

Comment:
This change looks like you're now setting the height of the wrong view. The height was being set on the inset view, not the fragment's view.